.github/workflows/publish.yml

```
@@ -16,10 +16,10 @@ jobs:
         python-version: '3.x'
     - name: Install dependencies
       run: |
-        python -m pip install --user --upgrade setuptools wheel
+        python -m pip install --user --upgrade setuptools wheel build
```

`build` installs what it needs, can omit these:
```suggestion
        python -m pip install --user --upgrade build
```

---

The only issue is that depending on the caching policy for isolated envs (which I can never recall precisely for different frontends), older versions that are installed by default may be used instead of the latest. I seem to remember this happening in the past, but maybe this isn't an issue nowadays...